state/src/impls/test_helper.rs

```diff
@@ -16,8 +16,10 @@
 
 #![macro_use]
 
+#[allow(unused)]
```

Comment:
These macros also should be moved into the module. I think it's good to remove them in these commits.

Reply:
Ok